pypeit/wavetilts.py

```diff
@@ -215,9 +215,8 @@ def __init__(self, mstilt, slits, spectrograph, par, wavepar, det=1, qa_path=Non
         # TODO -- Discuss further with JFH
         all_left, all_right, mask = self.slits.select_edges(initial=True, flexure=self.spat_flexure)  # Grabs all, initial slits
         # self.tilt_bpm = np.invert(mask == 0)
-        # We want to keep the 'BOXSLIT', for which mask value is 2. But we don't want to keep 'BOXSLIT'
-        # with other bad flag (for which the mask value would be > 2)
-        self.tilt_bpm = mask > 2
+        # we consider good slits 'BOXSLIT' (bpm=2), bad wavecalib (bpm=8) and the combination of the 2 (bpm=10)
+        self.tilt_bpm = (mask != 0) & (mask != 2) & (mask != 8) & (mask != 10)
```

Comment:
Again, use the bit arithmetic routines and bit names rather than hard wired bits. Same comment as above, i.e. we usually don't the tilts for box slits right? This may be something we added for DEIMOS to register the images (as we discussed before), but even then we need to distinguish between box slits where stars overlap and box slits where they don't which can be reduced.